web/viewer.js

```diff
@@ -1786,7 +1789,8 @@ window.addEventListener('updateviewarea', function () {
       'page': location.pageNumber,
       'zoom': location.scale,
       'scrollLeft': location.left,
-      'scrollTop': location.top
+      'scrollTop': location.top,
+      'pageRotation': PDFViewerApplication.pdfViewer.pagesRotation
```

Comment:
I think you should add `rotation: this._pagesRotation` at [pdf_viewer.js#L572](https://github.com/mozilla/pdf.js/blob/master/web/pdf_viewer.js#L572) instead, and change this line to `'rotation': location.rotation`.